lib/internal/Magento/Framework/File/Uploader.php

```diff
@@ -546,7 +546,7 @@ private function _setUploadFileId($fileId)
 
             preg_match("/^(.*?)\[(.*?)\]$/", $fileId, $file);
 
-            if (is_array($file) && count($file) > 0 && count($file[0]) > 0 && count($file[1]) > 0) {
+            if (is_array($file) && count($file) > 0 && !empty($file[0]) && (isset($file[1]) && !empty($file[1]))) {
```

Comment:
`isset($file[1])` is redundant here, please remove it.

Reply:
Hi @orlangur 
I removed redundant `isset($file[1])`